tasks/winbuildscripts/dobuild.bat

```diff
@@ -32,25 +37,21 @@ if "%TARGET_ARCH%" == "x64" (
     call ridk enable
 )
 
-set REPO_ROOT=%~p0\..\..
-pushd .
-cd %REPO_ROOT% || exit /b 101
-
 
-pip3 install -r requirements.txt || exit /b 102
+pip3 install -r requirements.txt || exit /b 120
```

Comment:
out of curiosity, why changing the error values?

Reply:
a [previous commit](https://github.com/DataDog/datadog-agent/pull/31466/commits/6e9e7404e902751b2e8c8c43271fe081f186ba95) added some lines with new errors to the .bat file

I later moved that code to python/omnibus, but didn't change the errors back. I think I'm going to make a follow-up PR to just replace the .bat file with either PowerShell or an invoke task.

Reply:
WIP: https://github.com/DataDog/datadog-agent/pull/31732